x/merkledb/node.go

```diff
@@ -66,17 +64,7 @@ func (n *node) hasValue() bool {
 
 // Returns the byte representation of this node.
 func (n *node) bytes() []byte {
-	if n.nodeBytes == nil {
```

Comment:
Removing this cached value means we'll have to reserialize the node before writing it. Are we concerned about the additional time that'll take? Seems to be a tradeoff between memory usage and CPU usage.

Reply:
anything we are writing is something we edited so this would have always been nil